scripts/package_building/build_artifacts.sh

```diff
@@ -523,8 +523,19 @@ function patch_kernel() {
     cp "$patches" "$kernel_path"
     pushd "$kernel_path"
     for patch in $(cat $patches); do
-        curl -s "$patch" | patch -f -p1 > /dev/null #2>&1
-        if [[ $? -ne 0 ]]; then
+        protocol=${patch%%://*}
+        url=${patch#*//}
+        case $protocol in
+            http|https)
+                curl -s "$patch" | patch -f -p1 > /dev/null
+                ;;
+            scp)
+                host=${url%%:*}
+                path=${url#*:}
+                ssh $host "cat $path" | patch -f -p1 > /dev/null
```

Comment:
use nohost check: -o StrictHostKeyChecking=no

otherwise it won't work the first time